src/App.tsx

```diff
@@ -1,26 +1,46 @@
+/* eslint-disable max-len */
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
-import { Loader, TodoFilter, TodoList, TodoModal } from './components';
 
-export const App = () => (
-  <>
-    <div className="section">
-      <div className="container">
-        <div className="box">
-          <h1 className="title">Todos:</h1>
+import { getTodos } from './api';
+import { TodoList } from './components/TodoList';
+import { TodoModal } from './components/TodoModal';
+import { TodoFilter } from './components/TodoFilter';
+import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { useAppDispatch, useAppSelector } from './app/hooks';
+import { todosSlice } from './features/todos';
 
-          <div className="block">
-            <TodoFilter />
-          </div>
+export const App: React.FC = () => {
+  const dispatch = useAppDispatch();
+  const currentTodo = useAppSelector(state => state.currentTodo);
```

Comment:
Consider adding a type for `state` in the `useAppSelector` hook to ensure type safety and improve code readability.